runtime/lua/vim/treesitter/query.lua

```diff
@@ -6,16 +6,23 @@ local language = require('vim.treesitter.language')
 --
 ---@class Query
 ---@field captures string[] List of captures used in query
----@field info table Contains used queries, predicates, directives
+---@field info QueryInfo Contains used queries, predicates, directives
```

Comment:
Maybe remove or at least split off the lines 4-5 above this? Lua-language-server will pick them up for hover on `Query` instead of the class definition.

(Or maybe improve it instead?)